randovania/games/prime1/logic_database/Chozo Ruins.txt

```diff
@@ -1886,55 +1903,97 @@ Extra - aabb: [225.01727, 76.59607, 54.75615, 343.10974, 209.55296, 138.8626]
 > In Arena (Front); Heals? False; Spawn Point; Default Node
   * Layers: default
   * Extra - world_position: (285.036987, 83.419861, 66.368805)
-  > Door to Sun Tower Access
-      After Flaahgra or After Chozo Ghosts (Sunchamber)
   > Door to Sunchamber Access
-      After Chozo Ghosts (Sunchamber)
+      After Flaahgra and After Chozo Ghosts (Sunchamber)
   > Event - Flaahgra
       Any of the following:
           All of the following:
-              Morph Ball
-              Any of the following:
-                  # Requirements for destroying the bomb slots
-                  Can Use Bombs
-                  Power Bomb ≥ 4 and Combat (Beginner) and Can Use Power Bombs
-              Any of the following:
-                  # Requirements for getting to the bomb slots
-                  Shoot Power Beam
-                  Missile ≥ 10 and Shoot Any Beam
-                  Space Jump Boots and Combat (Beginner) and L-Jump (Beginner)
-                  Combat (Intermediate) and Combat/Scan Dash (Advanced)
+              # Normal Fight
+              Morph Ball and Has Any Visor
               Any of the following:
                   # Requirements for shooting the first dish
                   Shoot Power Beam
-                  Missile and Shoot Any Beam
+                  Missile and Combat (Beginner) and Shoot Any Beam
+              Any of the following:
+                  # Requirements for reaching the Bomb Slots
+                  All of the following:
+                      # Shoot down the dishes to make the tentacles retract
+                      Any of the following:
+                          Shoot Power Beam
+                          All of the following:
+                              # Shoot dishes with only Missiles
+                              Combat (Beginner) and Shoot Any Beam
+                              Any of the following:
+                                  Missile ≥ 10
+                                  Missile ≥ 4 and Combat (Advanced)
+                      Any of the following:
+                          # Health requirements
+                          Combat (Intermediate) or Normal Damage ≥ 80
+                          Combat (Beginner) and Normal Damage ≥ 25
+                  All of the following:
+                      # Jump directly to the Bomb Slots. Dashing is done from dishes/tentacles
+                      Knowledge (Beginner)
```

Comment:
I argue that knowing that these 4 bomb slots are unlike every single other in the game in that they an be activated without actually slotting the ball inside them warrants knowledge intermediate. It's more knowledge than just knowing PBs work in addition to bombs, but it's less niche than knowing about wavebuster strategies.